cmake/stm32/devices.cmake

```diff
@@ -1095,12 +1095,18 @@ set(STM32_ALL_DEVICES
     L562RE
     L562VE
     L562ZE
-    MP151A
-    MP151C
-    MP153A
-    MP153C
-    MP157A
-    MP157C
+    MP151AA
+    MP151CA
+    MP151DA
+    MP151FA
+    MP153AA
+    MP153CA
+    MP153DA
+    MP153FA
+    MP157AA
+    MP157CA
+    MP157DA
+    MP157FA
```

Comment:
this is strange regarding datasheet but correct regarding the current STM32cmake implementation. This is also what made you use the A as size code... 
Is it possible to put MP151C instead MP151CA or will it break everything ?If it breaks, it's OK to leave as is.

Reply:
> If you feel like BSP will be fixed later then remove it from the tests so that it doesn't fail 
We probably need to remove the H7, WL and WB families as well then, they also fail in the same way.
The BSP test is not executed in the github actions.

> This is also what made you use the A as size code..
Yep :)

> Is it possible to put MP151C instead MP151CA or will it break everything ?
It will break because I used 
```cmake
set(STM32_MP1_TYPE_MATCH
     "MP151[AD]A.?" ...
```
But I don't mind at all changing to "MP151[AD](A.?)?", I agree that it'd be better.

Reply:
> We probably need to remove the H7, WL and WB families as well then, they also fail in the same way.

I didn't see they fail silently. So we can let them this way (and there is already #248 to remember it. Add info there if necessary)

> But I don't mind at all changing to MP151[AD](A.?)?, I agree that it'd be better.

If we both agree it would be better please go ahead :relaxed:

Reply:
Done!